app/views/lists/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="flex items-center justify-center p-12">
+  <div class="mx-auto w-full max-w-[550px]">
+    <%= simple_form_for @collection do |f| %>
+      <div class="mb-5">
+        <label for="name" class="mb-3 block text-base font-medium text-[#07074D]">
+          Книга
+        </label>
+        <%= f.label :book_ids, "Books" %>
```

Comment:
You are using Russian language almost in all labels. I don't see any sense to use English here

Reply:
Oh, thank you, I missed it